### PR TITLE
Multi journal interface

### PR DESCRIPTION
**Pull request by alfredodeza**
Accept only the scenario where journals are collocated and makes the interface change to be like:

    {"/device/osd": "/device/journal"}

Updates schemas, docs, and tests.

### CODE REVIEW

File: ceph_installer/tests/controllers/test_osd.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ def setup(self):
         data = dict(
             host="node1",
             fsid="1720107309134",
-            devices=['/dev/sdb'],
+            devices={'/dev/sdb': '/dev/sdc'},
             monitors=[{"host": "mon1.host", "interface": "eth1"}],
             journal_devices=["/dev/sdc"],
```
Comment (andrewschoen) [inline ceph_installer/tests/controllers/test_osd.py]: This should be removed, no?

File: ceph_installer/schemas.py [hunk 2]
```diff
@@ -37,7 +56,7 @@ def list_of_monitors(value):
 
 osd_configure_schema = (
     (optional("cluster_network"), types.string),
-    ("devices", list_of_devices),
+    ("devices", devices_object),
     ("fsid", types.string),
     ("host", types.string),
     ("journal_devices", list_of_devices),
```
Comment (andrewschoen) [inline ceph_installer/schemas.py]: This needs to be removed.

File: docs/source/index.rst [hunk 3]
```diff
@@ -423,7 +423,8 @@ Polling is not subject to handle state with HTTP status codes (e.g. 304)
           "monitors": [{"host": "mon0.host", "interface": "eth1"}],
       }
 
-   :<json array devices: (required) The devices to use for OSDs
+   :<json object devices: (required) A mapping of OSD device to Journal
+                          like device: {"device": "journal"}.
    :<json array journal_devices: (required) The devices to use for journal
```
Comment (andrewschoen) [inline docs/source/index.rst]: This needs to be removed here and in the above example

**Comment by andrewschoen** [comment]
test this please

**Comment by alfredodeza** [comment]
test this please

**Comment by alfredodeza** [comment]
this is getting ridiculous I swear the title is fixed now

**Pull request merged**